migrations/1528395641_add_user_id_to_campaign_plan.up.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
+
+ALTER TABLE campaign_plans
+    ADD COLUMN user_id integer,
+    ADD CONSTRAINT campaign_plans_user_id_fkey FOREIGN KEY (user_id)
+    REFERENCES users (id);
```

Comment:
How about adding this? That would allow us to, in some tests, defer the constraints until the end of the test transaction, which in turn allows us to use bogus IDs (`99999`) without having to create a user.

```suggestion
    REFERENCES users (id) DEFERRABLE INITIALLY IMMEDIATE;;
```